app/src/main/java/network/matic/maticsdkkotlinexample/MainActivity.kt

```diff
@@ -7,9 +7,10 @@ class MainActivity : AppCompatActivity() {
 
   override fun onCreate(savedInstanceState: Bundle?) {
     super.onCreate(savedInstanceState)
-
-    BalanceOfERC20.testBalanceOfERC20()
-    DepositEther.testDepositEther()
+      
+    SignTypedData.testSignTypedData()
```

Comment:
Why do you need to spin Android activity just to test library?

Reply:
cc @abhriyaroy

Reply:
@jdkanani I feel these are more like sample implementations than tests and hence the activity.